README.md

```diff
@@ -6,13 +6,34 @@ The purpose of this Nextflow workflow is to automate the process of indexing S3
 
 The benefits of using this workflows include:
 
-- There is a lack of data transfer and duplication because the S3 objects remain where they are.
+- Data duplication is avoided because the S3 objects remain where they are.
 - The folder-like structure in S3 is reproduced in Synapse to maintain the file organization.
 
+## Notes 
+
+### Caveat with computing MD5 checksums for Nextflow output files
+
+Nextflow does not compute MD5 checksums for published files. Hence, this workflow must compute these checksums post hoc. Unfortunately, there is on guarantee on the integrity of the file transfer from S3 to the EC2 instances that will compute the checksums. While the risk is minimized by remaining within the AWS network, it remains non-zero. 
+
+If you require high-fidelity MD5 checksums, make a feature request. This could be achieved by downloading each file more than once and ensuring consistency in the computed checksums.
```

Comment:
"This could be achieved by downloading each file more than once and ensuring consistency in the computed checksums."
Is this a suggestion for how one would accomplish high-fidelity MD5 checksums with this current version? 
If so, I would just add 
```suggestion
If you require high-fidelity MD5 checksums, make a feature request. Currently, this could be achieved by downloading each file more than once and ensuring consistency in the computed checksums.
```
for clarity

Reply:
This is an example of how we could implement it. I'll clarify the text.